doc/manual/manual.tex

```diff
@@ -2268,10 +2268,152 @@ \subsection{Particles}
+
+\begin{verbatim}
+docker run -it -v "$(pwd):/home/dealii/aspect/model_input:ro" \
+  gassmoeller/aspect:latest bash
+./aspect model_input/your_input_file.prm
+\end{verbatim}
+
+\item After the model has finished (or during the model run if you want to check intermediate results) copy the model output out of the container into your current directory. For this you need to find the name or ID of the docker container by running \texttt{docker ps -a} in a separate terminal first. Look for the most recently started container to identify your current \aspect{} container.
+
+\begin{verbatim}
+docker ps -a # Find the name of the running / recently closed container in the output
+docker cp CONTAINER_NAME:/home/dealii/aspect/model_output .
+\end{verbatim}
+
+\item The output data is saved inside your container even after the computation finishes and even when you stop the container. After you have copied out the data of the container you should therefore delete the container to avoid duplication of output data. Even after deleting you will always be able to start a new container from the downloaded image following step 2. Deleting the finished container can be achieved by the \texttt{docker container prune} command that removes any container that is not longer running (\textbf{Warning:} If you own other finished containers that you would like to keep only use \texttt{docker container rm CONTAINER\_NAME} to keep all other containers).
+
+\begin{verbatim}
+docker container prune
+\end{verbatim}
+or
+\begin{verbatim}
+docker container rm CONTAINER_NAME
+\end{verbatim}
+\end{enumerate}
+
+You are all set. Repeat steps 1-4 of this process as necessary when updating your model parameters. 
+
+\subsubsection{Developing \aspect{} within a container}
+
+The above given workflow does not include advice on how to modify \aspect{} inside the container. We recommend a slightly different workflow for advanced users that want to modify parts of \aspect{}. The \aspect{} docker container itself is build on top of a deal.II container that contains all dependencies for compiling \aspect{}. Therefore it is possible to run the deal.II container, mount an \aspect{} source directory from your host system and compile it inside of the container. An example workflow could look as following (assuming you navigated in a terminal into the modified \aspect{} source folder):
+
+\begin{verbatim}
+docker pull dealii/dealii:v8.5.pre.4-gcc-mpi-fulldepsmanual-debugrelease
+docker run -it -v "$(pwd):/home/dealii/aspect:ro" \
+  dealii/dealii:v8.5.pre.4-gcc-mpi-fulldepsmanual-debugrelease bash
+\end{verbatim}
+
+Inside of the container you now find a read-only \aspect{} directory that contains your modified source code. You can compile and run a model inside the container, e.g. in the following way:
+
+\begin{verbatim}
```

Comment:
like above, use the lstlisting environment. same for all other places here where you show command line commands.